fli-iam/shanoir-ng
# [fli-iam#1689] Check if linked acquisition before study card deletion

Test steps :
- Create study card
- Import dataset acquisitions set with this card
- When trying to delete the card from the list and from the details, a warning message ` The study-card XXX with id n is linked to other entities, it was not deleted.` is displayed
- Deletion of card with no linked acquisitions are still OK
